run/multiple_regions/main.tf

```diff
@@ -102,7 +102,7 @@ resource "google_compute_managed_ssl_certificate" "lb_default" {
   name     = "myservice-ssl-cert"
 
   managed {
-    domains = [var.domain_name]
+    domains = ["example.com"]
   }
 }
 # [END cloudrun_multiregion_cert]
```

Comment:
To fix the lint error, how about defining a temporary usecase

```suggestion
# [END cloudrun_multiregion_cert]

# Defined following to avoid unused variable lint error for `domain_name` variable
resource "local_file" "example" {
  content  = "${var.domain_name}\n"
  filename = "/tmp/example_text_file.txt"
}

```

Reply:
awesome suggestion, thank you!